Rakefile

```diff
@@ -449,7 +449,7 @@ NOTE: If you get the warning below, you can safely ignore it and move on to step
     $ gem update --system
           )
         ],
-        verify: -> { assert_version_is_sufficient('2.6.6', 'gem -v') }
+        verify: -> { assert_version_is_sufficient('2.6.8', 'gem -v') }
       },
 
       rvm: {
```

Comment:
Slack desktop client is under a new name and therefore fails the test https://github.com/ga-dc/installfest/pull/65/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR487

Should be: `/Applications/"Slack 2".app`